.travis.yml

```diff
@@ -29,6 +29,7 @@ jobs:
         - docker pull rocproject/travis-centos
         - docker pull rocproject/travis-archlinux
         - docker pull rocproject/travis-alpine
+        - docker pull rocproject/travis-opensuse
```

Comment:
Nitpick: please place opensuse right after centos in this file as well (here and below).